content/work/coachhub.md

```diff
@@ -0,0 +1,257 @@
+---
+title: 'Case Study: CoachHub'
+image:
+  src: projects/coachhub.jpg
+  position: top
+client:
+  name: &client ORCAS
```

Comment:
these &word pairings are curious ... but I'm going to guess this is done to be able to use some of the data throughout what you've built already (knowing we haven't built out page or case study templates yet?) or is this the way we need to enter some info now?

Reply:
this is a [feature of YAML](https://github.com/cyklo/Bukkit-OtherBlocks/wiki/Aliases-(advanced-YAML-usage)) that we haven't used much before, but it allows us to define something once, give it a name, and then re-use it somewhere else in the YAML. I started using it to add client information at the root level, while also attaching it to individual people (e.g. `venue: *client`) –  and to reference people (e.g. `&sara`) from multiple quotes (`<<: *sara`).